term/src/terminalstate/performer.rs

```diff
@@ -537,17 +537,13 @@ impl<'a> Performer<'a> {
                         .push(KeyboardEncoding::Kitty(flags));
                 }
             }
-            CSI::Keyboard(Keyboard::PushKittyState { flags, mode }) => {
+            CSI::Keyboard(Keyboard::PushKittyState(flags)) => {
                 if self.config.enable_kitty_keyboard() {
                     let current_flags = match self.screen().keyboard_stack.last() {
                         Some(KeyboardEncoding::Kitty(flags)) => *flags,
                         _ => KittyKeyboardFlags::NONE,
                     };
-                    let flags = match mode {
-                        KittyKeyboardMode::AssignAll => flags,
-                        KittyKeyboardMode::SetSpecified => current_flags | flags,
-                        KittyKeyboardMode::ClearSpecified => current_flags - flags,
-                    };
+                    let flags = current_flags | flags;
```

Comment:
I'm not totally confident about this part. To me the `screen_push_key_encoding_flags` function from Kitty is a litle cryptic so I'm not sure which mode it's emulating. I think it's merging with the top of the stack :sweat_smile: